comdb2/dbapi2.pyi

```diff
@@ -1,198 +0,0 @@
-# Copyright 2021 Bloomberg Finance L.P.
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#     http://www.apache.org/licenses/LICENSE-2.0
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-import datetime
-
-from typing import Any
-from typing import Callable
-from typing import Iterator
-from typing import List
-from typing import Mapping
-from typing import Optional
-from typing import Sequence
-from typing import Text
-from typing import Tuple
-from typing import Union
-
-from . import cdb2
-
-apilevel: str
-threadsafety: int
-paramstyle: str
-
-STRING: object
-BINARY: object
-NUMBER: object
-DATETIME: object
-ROWID: object
-
-
-def Binary(string: Union[Text, bytes]) -> bytes: ...
```

Comment:
This looks like the name of that function used to be different in the `pyi` vs the `py` file. I agree with your judgement that the `py` file should remain the same, and this is one more reason for folding the types back into the `py` file. Thanks for doing it again!

Reply:
No, not really - the `.py` file had:
```py
def _binary(string):
    ...

Binary = _binary
```
and the `.pyi` file had:
```py
def Binary(string: Union[Text, bytes]) -> bytes: ...
```

In order to merge those, I moved the annotations to the `_binary` function, but left `Binary` as an alias for that function. I could also flip those around, or drop `_binary` since it's private, or something. We could do
```py
def Binary(string: Union[Text, bytes]) -> bytes: ...
```
here and replace every usage of `_binary` with `Binary` if you prefer.

Reply:
It's fine either way, so let's leave it like this.